plugin/environment.go

```diff
@@ -151,14 +151,14 @@ func (env *Environment) Activate(id string) (manifest *model.Manifest, activated
 		return nil, false, fmt.Errorf("plugin not found: %v", id)
 	}
 
-	activePlugin := activePlugin{BundleInfo: pluginInfo}
+	plugin := activePlugin{BundleInfo: pluginInfo}
```

Comment:
I'm not sure about the naming, but I don't have any better option I think.

Reply:
I'd be in favour of renaming the struct to `activePluginStruct`. It's kinda ugly, but lets use use the more frequently used `activePlugin` where it matters, it matches the `*Func` pattern we use sometimes, and it's not exported so it "doesn't matter" ;P

Reply:
I don't like `activePluginStruct`. It is a struct, why do you need to state this in the name? 3/5

I'm fine with just using `plugin` as a variable name.

Reply:
@jespino What do you think?

Reply:
Yes, I don't love the `activePluginStruct` approach, but I don't have an strong opinion here.

Reply:
My objection to `plugin` is that -- in the context of this very function -- `plugin` is a `*model.BundleInfo`. `activePlugin` neatly distinguishes that, but of course sadly conflicts with the `struct` of the same name. In such a conflict, what's most important? The struct name or the variable name? I'd argue that if the struct is exported, then the struct name matters most, but if it's an internal abstraction, then the variable name matters more. So instead of renaming all the variable instances, we could just use `activePlugin<Something>`. 0/5 on the `<Something>`, 1/5 on renaming the variable.

I think `plugin` here, as you've changed it, is probably fine, but I don't feel like it's a general solution to the problem of internal naming clashes.